### ISSUE TITLE
Error Content-Type in headers: 'mime.lookup is not a function'

### ISSUE THREAD

**Issue by baileytalks** [orca-scan/express-md#3]
When running express-md for the first time I've got an unexpected error:

```
/Users/bailey/Projects/my-website/node_modules/express-md/index.js:111
var headers = { 'Content-Type': mime.lookup(ext) };
TypeError: mime.lookup is not a function
    at /Users/bailey/Projects/my-website/node_modules/express-md/index.js:111:58
    at /Users/bailey/Projects/my-website/node_modules/express-md/lib/memoize.js:38:34
```

I've tried passing the Content-Type using the headers option, and have also tried installing the mime node module. 

Any ideas?

PS I'm a junior level developer so if I've made a mistake, let me know. Here's the code: https://github.com/baileytalks/node-express-firebase-template/tree/error.

**Comment by jbreadner** [comment]
I've created a branch here that fixes the issue:

https://github.com/jbreadner/express-md/tree/issues/3

There's an open pull request to merge back to this repo.

**Comment by john-doherty** [comment]
Thanks again @jbreadner :) merged in and pushed 2.1.0 to npm 🙏🏽
